conf/cassandra.yaml

```diff
@@ -1302,6 +1302,10 @@ slow_query_log_timeout: 500ms
 #    Snitch for getting dc and rack of a node from metadata service of Alibaba cloud.
 #    This snitch that assumes an ECS region is a DC and an ECS availability_zone is a rack.
 #
+# AzureSnitch:
+#    Gets datacenter from 'location' and rack from 'zone' fields of 'compute' object
+#    from instance metadata service.
```

Comment:
If availability zone is not enabled it will use fault domains and get the respective value from